Documentation/user-guides/alerting.md

```diff
@@ -15,68 +15,29 @@ This guide assumes that you have a basic understanding of the Prometheus
 operator, and that you have already followed the [Getting Started]({{< ref
 "getting-started" >}}) guide.
```

Comment:
Should we mention that we also expect a running Alertmanager instance (explained in the Platform Guide's Getting Started)?

Reply:
Yeah agreed.